src/index.jsx

```diff
@@ -1,11 +1,38 @@
-import React from 'react';
-import { render } from 'react-dom';
-import './style.css';
+import React from "react";
+import { render } from "react-dom";
+import Footer from "./components/footer/Footer";
+import Header from "./components/header/Header";
+import ProductsSection from "./components/products-section/ProductsSection";
+import "./style.css";
+import products from "./db/products.js";
 
-const App = () => (
-  <>
-    <p>Tady asi něco bude</p>
-  </>
-);
+const App = () => {
+  const filterProducts = (category) => {
+    return products.filter((item) => item.category === category);
+  };
+  return (
+    <>
+      <div className="container">
+        <Header title="Móda" desc="Stylové oblečení a doplňky pro každý den" />
+        <ProductsSection
+          title="Šaty"
+          products={filterProducts("dresses")}
+          category="dresses"
+        />
```

Comment:
Luděk chtěl sice co kategorie to svoje vlastní komponenta, ale takhle udělané je to mnohem lepší. Tj mít i "generickou" komponentu "Section" které dáš title a produkty 👍 
I to žes dala data do vlastního souboru (db.js) a tady je filtruješ podle kategorie. Takhle by ti totiž došly z nějakého API. Až budeme brát useEffect hook tak to ještě trochu vylepšíme a budu po tobě chtít abys ta data z nějakého db.js vrátila jako výsledek asynchronní funkce, aby to bylo blíž k tomu jak reálně data do aplikace pak dojdou :)